packages/engine/Source/Shaders/Voxels/VoxelFS.glsl

```diff
@@ -103,14 +122,17 @@ void main()
             break;
         }
 
-        if (traversalData.stepT == 0.0) {
+        if (stepT == 0.0) {
             // Shape is infinitely thin, no need to traverse further
-            break;
+            // TODO: this doesn't happen? 
```

Comment:
Just a reminder that TODO comments should be removed before merging into `main`, if you weren't planning on doing so already 😄